snmp/tests/test_e2e_core_profiles/test_profile_audiocodes_mediant_sbc.py

```diff
@@ -0,0 +1,61 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .. import common
+from ..test_e2e_core_metadata import assert_device_metadata
+from .utils import (
+    assert_common_metrics,
+    assert_extend_generic_if,
+    assert_extend_generic_host_resources,
+    create_e2e_core_test_config,
+    get_device_ip_from_config,
+)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile_audiocodes_mediant_sbc(dd_agent_check):
+    config = create_e2e_core_test_config('audiocodes-mediant-sbc')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
+
+    ip_address = get_device_ip_from_config(config)
+    common_tags = [
+        'snmp_profile:audiocodes-mediant-sbc',
+        'snmp_host:audiocodes-mediant-sbc.device.name',
+        'device_namespace:default',
+        'snmp_device:' + ip_address,
+    ] + []
+
+    # --- TEST EXTENDED METRICS ---
+    # Examples:
+    assert_extend_generic_if(aggregator, common_tags)
+    assert_extend_generic_host_resources(aggregator, common_tags)
+
+    # --- TEST METRICS ---
+    assert_common_metrics(aggregator, common_tags)
+
+
+
+    # --- TEST METADATA ---
+    device = {
+        'description': 'audiocodes-mediant-sbc Device Description',
+        'id': 'default:' + ip_address,
+        'id_tags': ['device_namespace:default', 'snmp_device:' + ip_address],
+        'ip_address': '' + ip_address,
+        'name': 'audiocodes-mediant-sbc.device.name',
+        'profile': 'audiocodes-mediant-sbc',
+        'status': 1,
+        'sys_object_id': '1.1.2.3.5.8.13.21.34',
```

Comment:
I don't get how the test passed and got the correct profile since this sysoid is not correct ?